src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -139,18 +140,36 @@ julia> vertices(square)
  [-1, 1]
  [1, 1]
 
-julia> nc = normal_cone(square, 1)
+julia> nc = normal_cone(square, [1])
 Polyhedral cone in ambient dimension 2
 
 julia> rays(nc)
 2-element SubObjectIterator{RayVector{QQFieldElem}}:
  [1, 0]
  [0, 1]
-```
+
+ 
+ julia> C = normal_cone(P, [1,2])
+Polyhedral cone in ambient dimension 2
+
+julia> facets(C)
+1-element SubObjectIterator{LinearHalfspace{QQFieldElem}} over the halfspaces of R^2 described by:
+-x_2 <= 0
+
+
+julia> C = normal_cone(P, [1,2], outer = true)
+Polyhedral cone in ambient dimension 2
+
+julia> facets(C)
+1-element SubObjectIterator{LinearHalfspace{QQFieldElem}} over the halfspaces of R^2 described by:
+x_2 <= 0
+
+
+ ```
 """
-function normal_cone(P::Polyhedron{T}, i::Int64) where {T<:scalar_types}
-  @req 1 <= i <= n_vertices(P) "Vertex index out of range"
-  bigobject = Polymake.polytope.normal_cone(pm_object(P), Set{Int64}([i - 1]))
+function normal_cone(P::Polyhedron{T}, F::Vector{Int64}; outer::Bool = false) where {T<:scalar_types}
+  @req 1 <= length(F) <= n_vertices(P) "Vertex index out of range"
+  bigobject = Polymake.polytope.normal_cone(pm_object(P), Set{Int64}(F .- 1), outer=Int64(outer))
   return Cone{T}(bigobject, coefficient_field(P))
 end
```

Comment:
As mentioned above, we do not want to break the API. So please add the old function signature as well:
```suggestion
end
normal_cone(P::Polyhedron{T}, i::Int64) where {T<:scalar_types} = normal_cone(P, [i])
```
or something similar.